apps/node/tests/test19.ts

```diff
@@ -0,0 +1,25 @@
+import { Assets } from '..';
```

Comment:
This is a good test, but please add the logic to one of the existing form filling tests. The integration tests take a lot of time to run, so we try to keep the number of files to a minimum.

Reply:
Also, please be sure to add the same logic to the `deno`, `web`, and `rn` tests. The tests should always be identical across each runtime/environment. Note that you may not be able to actually run the `rn` tests. That's fine, but please still update the file. I'll run it along with the others before releasing the changes.